modules/dnn/CMakeLists.txt

```diff
@@ -92,9 +92,15 @@ endif()
 
 set(dnn_runtime_libs "")
 if(INF_ENGINE_TARGET)
+  ocv_option(OPENCV_DNN_IE_NN_BUILDER_2019 "Build with Inference Engine NN Builder API support" ON)
```

Comment:
Why not OFF by default?

Reply:
Current patch state helps to ensure that we don't lost anything (accuracy / performance).

I will switch defaults in another small commit including backend value for IE.